pkgs/by-name/pa/paperlib/package.nix

```diff
@@ -0,0 +1,57 @@
+{ lib
+, stdenv
+, fetchurl
+, pkgs
+, appimageTools
+}:
```

Comment:
AFAICT, this file is still not formatted according to the style specified by [*Nix RFC 166*](https://github.com/NixOS/rfcs/blob/master/rfcs/0166-nix-formatting.md#reformat-nixpkgs), which is a style that all `.nix` files in Nixpkgs must adhere to.

The following is quoted from *Nix RFC 166* section [*Reformat Nixpkgs*](https://github.com/NixOS/rfcs/blob/master/rfcs/0166-nix-formatting.md#reformat-nixpkgs):
> For formatting Nixpkgs itself, a pinned release version of the official Nix formatter must be used. CI must generally enforce all Nix files in Nixpkgs to be formatted with this version at all times. Automatically generated files may be handled differently, see [this sub-section](https://github.com/NixOS/rfcs/blob/master/rfcs/0166-nix-formatting.md#automatically-generated-files).

To ensure that the Nix expression in this file follows the *Nix RFC 116*, please format it with `pkgs.nixfmt-rfc-style`:
```bash
nix run .#nixfmt-rfc-style -- pkgs/by-name/pa/paperlib/package.nix
```